MYshort.bib

```diff
@@ -155,50 +182,66 @@ @String{SAC
 @String{SBES = "SBES"}
 @String{SBLP = "SBLP"}
 @String{SC = "SC"}
+@String{SOSP = "SOSP"}
 @String{SSR = "SSR"}
 @String{SST = "SST"}
 @String{VLHCC = "VL/HCC"}
 
 journal
 @String{AUSE = "AUSE"}
 @String{CACM = "Comm.\ ACM"}
+@String{CCPE = "CCPE"}
+@String{CIM = "CIM"}
+@String{COLA = "COLA"}
 @String{ComNet = "ComNet"}
 @String{Computer = "IEEE Computer"}
 @String{CSUR = "CSUR"}
 @String{EMSE = "EMSE"}
+@String{EXWA = "EXWA"}
 @String{FAC = "FAC"}
 @String{FMSD = "FMSD"}
+@String{IBMSJ = "IBM Syst. J."}
 @String{IC = "IC"}
+@String{IJAST = "IJAST"}
 @String{IJCSI = "IJCSI"}
+@String{IJPE = "IJPE"}
 @String{IJSEKE = "IJSEKE"}
 @String{IS = "IS"}
 @String{IST = "IST"}
-@String{JBCS = "JBCS"}
 @String{JACM = "J.\ ACM"}
 @String{JAR = "JAR"}
+@String{JAIR = "JAIR"}
 @String{JBCS = "JBCS"}
 @String{JLAMP = "JLAMP"}
 @String{JLAP = "JLAP"}
+@String{JOM = "JOM"}
+@String{JOOP = "JOOP"}
 @String{JOT = "JOT"}
+@String{JR = "JR"}
+@String{JRCS = "JRCS"}
 @String{JSEP = "JSEP"}
 @String{JSS = "JSS"}
 @String{JUCS = "J.UCS"}
 @String{NJC = "NJC"}
 @String{OSR = "OSR"}
+@String{PPL = "PPL"}
+@String{PVLDB = "PVLDB"}
 @String{REJ = "REJ"}
 @String{SCP = "SCP"}
 @String{SEN = "SEN"}
 @String{SIGPLAN = "SIGPLAN Not."}
 @String{SME = "SME"}
 @String{JSME = "JSME"}
 @String{Software = "IEEE Software"}
+@String{SOSYM = "SoSyM"}
```

Comment:
Again, there should be only one consistent capitalization. Ideally, this is also consistent with the capitalization found in the web.

Reply:
removed duplicates now